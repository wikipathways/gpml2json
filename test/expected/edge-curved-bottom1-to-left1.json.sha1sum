9324c2661e06caa6b9dbb0f818c39df18026fc1e ?-
