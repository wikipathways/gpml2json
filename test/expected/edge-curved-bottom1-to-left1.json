{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["f1448","cf394","fc9ba"],"drawAs":"rect","gpmlElementName":"Pathway","height":148.77800149472392,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":233.301861197194,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/349f62aaef289b60450f468b82f5736cfb59568ac57018f172a368124f092bb5"},"entityMap":{"cf394":{"id":"cf394","textContent":"GeneProduct","wpType":"GeneProduct","x":123.301861197194,"y":98.77800149472392,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32993,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"fc9ba":{"id":"fc9ba","textContent":"0015","wpType":"GeneProduct","x":13.356737782051738,"y":51.14655258141399,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32994,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"f1448":{"id":"f1448","drawAs":"CurvedLine","borderWidth":1,"zIndex":32992,"points":[{"isAttachedTo":"fc9ba","attachmentDisplay":{"position":[0.5,1],"offset":[0,0]},"x":53.35673778205174,"y":71.146552581414,"orientation":[0,1]},{"isAttachedTo":"cf394","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":123.301861197194,"y":108.77800149472392,"orientation":[1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["fc9ba","cf394"]}}}
